packages/miniflare/src/workers/shared/remote-proxy-client.worker.ts

```diff
@@ -23,4 +25,24 @@ export default class Client extends WorkerEntrypoint<{
 
 		return fetch(this.env.remoteProxyConnectionString, req);
 	}
+
+	constructor(ctx: ExecutionContext, env: Env) {
+		const url = new URL(env.remoteProxyConnectionString);
+		url.protocol = "ws:";
+		url.searchParams.set("MF-Binding", env.binding);
+		const session = rpcOverWebSocket(url.href);
+		const stub = session.getStub();
+
+		super(ctx, env);
+
+		return new Proxy(this, {
+			get(target, prop) {
+				if (Reflect.has(target, prop)) {
+					return Reflect.get(target, prop);
+				}
+
+				return Reflect.get(stub, prop);
```

Comment:
could you explain this logic here? 🙏 

does it mean that jsrpc has some limitations/doesn't stub everything?

Reply:
jsrpc stubs everything, but we don't want to stub anything that's defined directly in the proxy worker. In this case it's just `fetch()` (which doesn't need to go over JSRPC and is more efficient as a direct fetch call). We could check `prop === "fetch"`, but `Reflect.has()` seemed nicer and more flexible